draft-xyz-rats-psa-endorsements.md

```diff
@@ -131,94 +140,149 @@ firmware measurement associated with Implementation ID
 ~~~
 {: #ex-reference-value title="Example Reference Value"}
 
-### Firmware Updates and Patches
-{: #sec-fw-evo}
+## Identity
 
-Firmware RoT descriptors that are part of the same upgrade chain can be linked
-together using one of `comid-patches` or `comid-updates` relations, depending on
-the precise nature of their relationship.  For example, if using semantic
-versioning {{SEMA-VER}}, a bump in the MINOR version number would be associated
-with a `comid-updates` relation, whereas an increase in the PATCH indicator
-would have a corresponding `comid-patches` relation between the involved
-components.  Note that `comid-updates` relations would only occur between
-firmware RoT descriptors that have PATCH number 0 and "adjacent" MINOR numbers
-as illustrated in {{fig-updates-patches}}.  Note that changing the MAJOR
-version number would typically result in a separate product / upgrade chain.
+{: #sec-identity}
+
+Identity Claims carry the verification key associated with the Initial
+Attestation Key (IAK) of a PSA device.
+
+Each verification key is encoded alongside the corresponding device Instance ID
+in a CoMID identity claim.
+
+The Instance ID is encoded using `tagged-ueid` as `$device-id-type`.
+
+The IAK public key is encoded as a COSE Key according to Section 7 of
+{{!RFC8152}} and wrapped in the `COSE_KeySet`. The number of items in the
+`COSE_KeySet` MUST be 1.
+
+The example in {{ex-identity-claim}} shows a CoMID identity claim carrying a
+secp256r1 EC public IAK associated with Instance ID `4ca3...d296`.
 
-~~~ goat
-{::include art/updates-patches.txt}
 ~~~
-{: #fig-updates-patches title="Updates, Patches Relations and Semantic Versioning"}
+{::include examples/instance-pub.diag}
+~~~
+{: #ex-identity-claim title="Example Identity Claim"}
 
-The Reference Value CoMID of the patching or updating firmware has a
-`linked-tag-entry` map populated as follows:
+### CoMID Layout
 
-* key 0 contains the tag identifier of the Reference Value CoMID of the patched
-  or updated firmware;
-* key 1 contains one of `comid-patches` or `comid-updates` relations.
+An Identity CoMID can carry as many Identity Claims as needed.
 
-{{ex-linked-tag}} provides an example of a Reference Value CoMID patching
-another Reference Value CoMID with a UUID tag identifier
-`3f06af63-a93c-11e4-9797-00505690773f`.
+## Certification
+
```

Comment:
the ref anchor must be attached to the element it refers to
```suggestion
```

Reply:
This was giving a warning that Headings should be surrounded by blank lines!